common/thorhelper/roxiehelper.cpp

```diff
@@ -2399,7 +2399,7 @@ void ClusterWriteHandler::getPhysicalName(StringBuffer & name, const char * clus
 {
     Owned<IStoragePlane> plane = getDataStoragePlane(cluster, false);
     const char * prefix = plane ? plane->queryPrefix() : nullptr;
-    makePhysicalPartName(logicalName.get(), 1, 1, name, 0, DFD_OSdefault, prefix);
+    makePhysicalPartName(logicalName.get(), 1, 1, name, 0, DFD_OSdefault, prefix, false);
```

Comment:
Shouldn't this be passing in dirPerPart from the plane/default, or do roxie/hthor always not create a subdir?  If so it would be worth commenting.

Reply:
The feature is disabled for 1 part files, i.e. a file will only be published with this flag if it has >1 part.